### PR TITLE
Add desc of "Storage Write Stall" row in the TiFlash grafana

### PR DESCRIPTION
**Pull request by JaySon-Huang**
Signed-off-by: JaySon-Huang <tshent@qq.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/6524
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/monitor-tiflash.md [hunk 1]
```diff
@@ -70,6 +70,13 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
 >
 > 目前这部分监控指标仅包含了 TiFlash 存储层的统计指标，未包括 TiFlash-Proxy 内的信息。
 
+## Storage Write Stall
+
+- Write & Delta Management Throughput: 所有实例的写入及数据整理的吞吐量。`throughput_write` 意味着通过 Raft 进行数据同步的吞吐量。`throughput_delta-management` 是指数据整理的吞吐量。`total_write` 是指自上次启动以来的总写入字节数。`total_delta-management` 是指自上次启动以来数据整理的总字节数。
+- Write Stall Duration: 每个实例写入和移除 Region 数据发生 stall 的时长。
+- Write Throughput By Instance: 每个实例写入数据的吞吐量。它包括 apply Raft 数据日志以及 Raft 快照的写入吞吐量。
+- Write Command OPS By Instance: 每个实例收到的各种命令的总计数。`write block` 代表着通过 Raft 同步数据日志。`delete_range` 表示一些 Region 从该实例中删除或移动到该实例中。`ingest` 表示着一些 Region 的快照被 apply 到这个实例中gco -b add_tiflash_metrics_write_stall。
```
Comment (JaySon-Huang) [inline tiflash/monitor-tiflash.md:78]:
```suggestion
- Write & Delta Management Throughput：所有实例的写入及数据整理的吞吐量。`throughput_write` 意味着通过 Raft 进行数据同步的吞吐量。`throughput_delta-management` 是指数据整理的吞吐量。`total_write` 是指自上次启动以来的总写入字节数。`total_delta-management` 是指自上次启动以来数据整理的总字节数。
- Write Stall Duration：每个实例写入和移除 Region 数据发生 stall 的时长。
- Write Throughput By Instance：每个实例写入数据的吞吐量。它包括 apply Raft 数据日志以及 Raft 快照的写入吞吐量。
- Write Command OPS By Instance：每个实例收到的各种命令的总计数。`write block` 代表着通过 Raft 同步数据日志。`delete_range` 表示一些 Region 从该实例中删除或移动到该实例中。`ingest` 表示着一些 Region 的快照被 apply 到这个实例中。
```

**Comment by TomShawn** [comment]
@JaySon-Huang Could you please involve one or two technical reviewers? Thanks!

File: tiflash/monitor-tiflash.md [hunk 2]
```diff
@@ -70,6 +70,13 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
 >
 > 目前这部分监控指标仅包含了 TiFlash 存储层的统计指标，未包括 TiFlash-Proxy 内的信息。
 
+## Storage Write Stall
+
+- Write & Delta Management Throughput：所有实例的写入及数据整理的吞吐量。`throughput_write` 意味着通过 Raft 进行数据同步的吞吐量。`throughput_delta-management` 是指数据整理的吞吐量。`total_write` 是指自上次启动以来的总写入字节数。`total_delta-management` 是指自上次启动以来数据整理的总字节数。
```
Comment (TomShawn) [inline tiflash/monitor-tiflash.md:75]:
```suggestion
- Write & Delta Management Throughput：所有实例上的数据写入及数据整理的吞吐量。
- `throughput_write` 表示通过 Raft 进行数据同步的吞吐量。
- `throughput_delta-management` 表示数据整理的吞吐量。
- `total_write` 表示自上次启动以来的总写入字节数。
- `total_delta-management` 表示自上次启动以来数据整理的总字节数。
```

File: tiflash/monitor-tiflash.md [hunk 3]
```diff
@@ -70,6 +70,13 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
...
+- Write Stall Duration：每个实例写入和移除 Region 数据发生 stall 的时长。
+- Write Throughput By Instance：每个实例写入数据的吞吐量。它包括 apply Raft 数据日志以及 Raft 快照的写入吞吐量。
```
Comment (TomShawn) [inline tiflash/monitor-tiflash.md:77]:
```suggestion
- Write Throughput By Instance：每个实例写入数据的吞吐量，包括 apply Raft 数据日志以及 Raft 快照的写入吞吐量。
```

File: tiflash/monitor-tiflash.md [hunk 4]
```diff
@@ -70,6 +70,13 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
...
+- Write Command OPS By Instance：每个实例收到的各种命令的总计数。`write block` 代表着通过 Raft 同步数据日志。`delete_range` 表示一些 Region 从该实例中删除或移动到该实例中。`ingest` 表示着一些 Region 的快照被 apply 到这个实例中。
```
Comment (TomShawn) [inline tiflash/monitor-tiflash.md:78]:
```suggestion
- Write Command OPS By Instance：每个实例收到各种命令的总计数。
    - `write block` 表示通过 Raft 同步数据日志。
    - `delete_range` 表示从该实例中删除一些 Region 或移动一些 Region 到该实例中。
    - `ingest` 表示这些 Region 的快照被 apply 到这个实例中。
```

**Comment by flowbehappy** [comment]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

File: tiflash/monitor-tiflash.md [hunk 5]
```diff
@@ -70,6 +70,20 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
...
 > 目前这部分监控指标仅包含了 TiFlash 存储层的统计指标，未包括 TiFlash-Proxy 内的信息。
 
+## Storage Write Stall
+
+- Write & Delta Management Throughput：所有实例写入及数据整理的吞吐量。
+    - `throughput_write` 表示通过 Raft 进行数据同步的吞吐量。
+    - `throughput_delta-management` 表示数据整理的吞吐量。
+    - `total_write` 表示自上次启动以来的总写入字节数。
+    - `total_delta-management` 表示自上次启动以来数据整理的总字节数。
+- Write Stall Duration：每个实例写入和移除 Region 数据发生 stall 的时长。
```
Comment (shichun-0415) [inline tiflash/monitor-tiflash.md:80]:
```suggestion
- Write Stall Duration：每个实例写入和移除 Region 数据产生的卡顿时长。
```

File: tiflash/monitor-tiflash.md [hunk 6]
```diff
@@ -70,6 +70,20 @@ TiFlash 面板一共包括 **TiFlash-Summary**、**TiFlash-Proxy-Summary**、**T
...
+- Write Throughput By Instance：每个实例写入数据的吞吐量，包括 apply Raft 数据日志以及 Raft 快照的写入吞吐量。
+- Write Command OPS By Instance：每个实例收到各种命令的总计数。
+    - `write block` 表示通过 Raft 同步数据日志。
+    - `delete_range` 表示从该实例中删除一些 Region 或移动一些 Region 到该实例中。
+    - `ingest` 表示这些 Region 的快照被 apply 到这个实例中。
```
Comment (shichun-0415) [inline tiflash/monitor-tiflash.md:85]:
```suggestion
    - `ingest` 表示这些 Region 的快照被应用到这个实例中。
```

**Pull request merged**